### DISCUSSION TITLE
Does copilot working in vs code now?

### DISCUSSION THREAD

**Discussion by effortprogrammer** [community/community#6971] [Copilot Conversations]
I used copilot in vs code properly yesterday, but not working now in my vs code. So, I wondered if anyone else having the same issue.

**Comment by davecheney** [comment]
@effortprogrammer could you please describe the symptoms.

**Reply by effortprogrammer** [reply]
Hi, @davecheney  It seems copilot is installed and running globally, but it didn't show any suggestion, which is pretty different from yesterday. Yesterday, the suggestion was pretty fast and vary, but now, nothing was suggested by the copilot. I tried java, python, js, but none of them worked.

**Reply by davecheney** [reply]
@effortprogrammer I see that most of your completions were cancelled before returning a result -- you typed some more so copilot started a new completion request -- I can also see some timeouts but its not clear why copilot didn't return an answer. I don't have anything else to offer at the moment I'm afraid.

**Reply by effortprogrammer** [reply]
@devoncurry So, what should I do with this problem? Do you think removing the copilot extension and re-install can solve this problem?

**Reply by davecheney** [reply]
@effortprogrammer reinstalling won't help, copilot just has nothing to suggest at the moment.

**Reply by effortprogrammer** [reply]
@davecheney So, what you're saying is there's no wrong with the copilot, so write more code for a better suggestions?

**Reply by davecheney** [reply]
Yup, copilot is up and working, if copilot doesn't have anything to suggest at the moment please don't take it personally.
